packages/rest-api/src/controllers/bridgeController.ts

```diff
@@ -1,26 +1,39 @@
 import { validationResult } from 'express-validator'
 import { parseUnits } from '@ethersproject/units'
+import { isAddress } from 'ethers/lib/utils'
 
 import { formatBNToString } from '../utils/formatBNToString'
 import { Synapse } from '../services/synapseService'
+import { tokenAddressToToken } from '../utils/tokenAddressToToken'
 
 export const bridgeController = async (req, res) => {
   const errors = validationResult(req)
   if (!errors.isEmpty()) {
     return res.status(400).json({ errors: errors.array() })
   }
   try {
-    const { fromChain, toChain, amount } = req.query
-    const fromTokenInfo = res.locals.tokenInfo.fromToken
-    const toTokenInfo = res.locals.tokenInfo.toToken
+    const { fromChain, toChain, amount, fromToken, toToken } = req.query
+
+    if (!isAddress(fromToken) || !isAddress(toToken)) {
```

Comment:
if this is already checked in the route, it isn't needed here since the validations happen upstream in `bridgeRoute`